cfme/tests/test_db_migrate_manual.py

```diff
@@ -588,3 +588,59 @@ def test_update_webui_replication():
         testtype: functional
     """
     pass
+
+
+@pytest.mark.manual
+@test_requirements.update
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1678337])
+def test_embedded_ansible_update():
+    """ Check Embedded ansible role start successfully after upgrade
+
+    Bugzilla:
+        1678337
+    Polarion:
+        assignee: jhenner
+        casecomponent: Configuration
+        caseimportance: medium
+        initialEstimate: 1h
+        testSteps:
+            1. Provision appliance
+            2. Subscribe to target version channels and update the packages
+            3. Run rake db:migrate command and wait for it complete
```

Comment:
Q: I'm suspect db migration is part of update or upgrade? If upgrade i would try to give receptive name.